http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -82,7 +84,19 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
 
       enableIntraNodeVisibility = req.labels.get(AOU_UI_LABEL).isDefined
       _ <- req.appType match {
-        case AppType.Galaxy | AppType.HailBatch | AppType.Wds | AppType.RStudio | AppType.Cromwell => F.unit
+        case AppType.Galaxy | AppType.HailBatch | AppType.Wds | AppType.Cromwell => F.unit
+        case AppType.Allowed =>
+          req.allowedChartName match {
+            case Some(cn) =>
+              cn match {
+                case AllowedChartName.RStudio => F.unit
+                case AllowedChartName.Sas     => F.unit // TODO: check group permission check here
```

Comment:
Does one need to be in a special group in order to launch SAS?

Reply:
yea...it's not implemneted in this PR since SAS won't get enabled yet...but it will be becuz for development purpose, only ppl who have SAS license can access it